documentation/modules/cruise-control/ref-cruise-control-configuration.adoc

```diff
@@ -103,6 +108,12 @@ metadata:
   name: my-cluster
 spec:
   # ...
+  kafka:
+    # ...
+    resources:
+      limits:
```

Comment:
Should this be request rather than limit?

Reply:
This sets requests == limits.

Reply:
Yes, but it is not clear to everyone. Setting only request also means that it will be the limit but it is much more clear what it means to everyone.

Reply:
Ok.